WordPressKit/WordPressComRestApi.swift

```diff
@@ -421,25 +456,6 @@ extension WordPressComRestApi {
     @objc class public func anonymousApi(userAgent: String) -> WordPressComRestApi {
         return WordPressComRestApi(oAuthToken: nil, userAgent: userAgent)
     }
-
-    /// Append the user's preferred device locale as a query param to the URL path.
-    /// If the locale already exists the original path is returned.
-    ///
-    /// - Parameters:
-    ///     - path: A URL string. Can be an absolute or relative URL string.
-    ///
-    /// - Returns: The path with the locale appended, or the original path if it already had a locale param.
-    ///
-    @objc class public func pathByAppendingPreferredLanguageLocale(_ path: String) -> String {
```

Comment:
Do we no longer need to use this method on ObjC in the main app?

Reply:
I search for consumers of this in the `try/wordpresskit-72-validation` branch I created, and could not find any.